elvee.c

```diff
@@ -303,7 +304,31 @@ int main(int argc, char **argv)
 
 #ifdef WINDOWS
 
+    // Quote arguments if necessary and track those quoted.
+
+    char **qargv = NULL;
+    for (int i = 0; i < argc; i++) {
+        char *qarg = argv_quote(argv[i]);
+        if (qarg != argv[i]) {
+            if (qargv == NULL) {
+                qargv = calloc(argc, sizeof(char*));
+            }
+            qargv[i] = qarg;
+        }
+        argv[i] = qarg;
+    }
+
     intptr_t result = _spawnv(_P_WAIT, spawn_path, argv);
+
+    // Free any quote arguments, including their tracking.
+
+    if (qargv) {
+        for (int i = 0; i < argc; i++) {
+            free(qargv[i]);
+        }
+        free(qargv);
+    }
```

Comment:
I wonder, if the idea was to release rhe memory for code maintenance reasons, I assume there is a chance that you'll add code after this point? Is it wise to leave argv with dangling pointers here?

Reply:
That's a good point. I think it would be fair to leave a comment at this point (specifically) since it's a lot more obvious here that the program ends right after that a future maintainer would need to address should some post-processing be added. At the same time, it would be equally simple to just take care of it. Can I leave the decision with you and follow-up with the change? Thanks!

Reply:
There you go.